bundle/config/mutator/default_target.go

```diff
@@ -13,11 +13,13 @@ type defineDefaultTarget struct {
 	name string
 }
 
+const DefaultTargetPlaceholder = "default"
```

Comment:
This variable does not seem to be used anywhere besides that one place below, why was it extracted (and exported)?

Reply:
Good catch, this is leftover from something I was trying out. Reverted.